projects/plugins/boost/changelog/add-boost-interstitial-modal-critical-css

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Jetpack Boost: add product interstitial modal for boost
```

Comment:
```suggestion
UI: upgrade CTA now opens a modal instead of navigating to the upgrade page.
```

Reply:
Small suggestion, since these entries go straight into the changelog.md and readme.txt when releasing.